pkg/agent/controller/traceflow/packetin.go

```diff
@@ -298,3 +304,16 @@ func getNetworkPolicyObservation(tableID uint8, ingress bool) *opsv1alpha1.Obser
 	}
 	return ob
 }
+
+func isValidCtNw(ipStr string) bool {
+	ip := net.ParseIP(ipStr)
+	if ip == nil {
+		return false
+	}
+	// Reserved by IETF	[RFC3513][RFC4291]
```

Comment:
Not sure why a tab is used between `IETF` and `[RFC3513]` rather than a space. Is it a convention?
There are quite some reserved IP blocks. Why only this one is checked?
https://www.iana.org/assignments/ipv6-address-space/ipv6-address-space.xhtml

Reply:
It's IDE issue. In my IDE, there is only one space here. Will fix it.
When OVS processing ICMPv6 packet, we'll get conntrack IPv6 inside this reserved IP block although there is no DNAT. Not sure why this occur in OVS. This change is to skip this IPv6 block to avoid unexpected LB observation.